awesome_owl/static/src/todo_list/todo_item.js

```diff
@@ -0,0 +1,27 @@
+import { Component } from "@odoo/owl";
+
+export class TodoItem extends Component {
+    static template = "awesome_owl.TodoItem";
+    static props = {
+        todo: {
+            type: Object,
+            shape: {
+                id: Number,
+                description: String,
+                isCompleted: Boolean,
+            },
+        },
+        toggleState: Function,
+        delete: Function,
+    };
+    
+    onChange() {
+        if (this.props.toggleState) {
+            this.props.toggleState(this.props.todo.id);
+        }
```

Comment:
Time to use the cool js feature again

```suggestion
        this.props.toggleState?.(this.props.todo.id);
```